lib/elixir/lib/macro.ex

```diff
@@ -370,6 +370,27 @@ defmodule Macro do
       iex> Macro.escape({:unquote, [], [1]}, unquote: true)
       1
 
+  ## Comparison to `Kernel.quote/2`
+
+  The `escape/2` function is sometimes confused with `Kernel.SpecialForms.quote/2`,
+  because the above examples behave the same with both. The key difference is
+  best illustrated when the value to escape is stored in a variable.
+
+      iex> quote do: {:a, :b, :c}
+      {:{}, [], [:a, :b, :c]}
+
```

Comment:
I think we are missing the ` iex> Macro.escape({:a, :b, :c})` example

Reply:
I believe I left it out intentionally, since it's mentioned in the first set of examples. Do you feel like it's worth repeating here?

Reply:
I dont think it will hurt being explicit in this particular case

Reply:
...but if you want to leave it out it's still fine

Reply:
I added it!